app/buy/index.js

```diff
@@ -1,69 +1,100 @@
+/* eslint-disable no-undef */
```

Comment:
Instead of disabling this for the file, let's try to make ESLint smarter. You can set `globals` in the `.eslintrc.json` file, as shown here: https://eslint.org/docs/latest/use/configure/language-options#specifying-globals

Try to see if you can specify `require` as a global, in order to get rid of this comment on line 1 (and also the one on line 53).